### ISSUE TITLE
Review the top-level teams and rearrange as appropriate

### ISSUE THREAD

**Issue by ehuss** [rust-lang/leadership-council#32]
There are some open questions about what the appropriate set of top-level teams should be for the project. Some things to consider:

* Find permanent homes for launching pad teams.
* Consider if there should be some balance between sizes of top-level teams.
* Try to manage the number of representatives on the council (preferred 5 to 8).
* Consider where new teams might get created that don't fit within the categories of the existing top-level teams. For example, let's say we have a "docs" team, where might that go?
* Ensure that top-level teams have well-documented purviews, starting to identify gaps between those purviews, and working with teams to determine when those gaps should fall to specific existing teams or become the purview of new teams.

I don't offhand know what other considerations there might be, but I assume there are several more.

**Comment by traviscross** [comment]
As this is marked as active but has not seen an update here for some time, I nominate to review this and to consider whether we should leave an update here about the status.

**Comment by ehuss** [comment]
I'm going to close this issue as I think we have finished some of these items, and are splitting others out into separate issues.

* Opened #118 specifically for the launching pad.
* We have reorganized the top-level teams over the past year (crates.io to devtools, and release to infra). For the most part, I think we are generally happy with the size, but I think we'll expect the number and structure to change continually over time as the project evolves, and we don't need an issue to track that.
* We don't have specific guidelines on how to decide the formation of new top-level teams (particularly ones filling the gaps). I suspect we will learn this as we go along, and after going through the process we might formulate guidelines we'll want to write down.
* #44 will work on charters which should help clarify the top-level team purviews. Similar to the point above, I think we will continue to evolve around the gaps as we work on any restructuring or new team proposals.
